src/games/tic_tac_toe.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+exports.setUp = function setUp(players) {
+  let player_sides = {
+    [players[0]]: 'X',
+    [players[1]]: 'O'
+  };
+  let board = [];
+  for (let x = 0; x < 3; y++) {
+    board[x] = [];
+    for (let y = 0; y < 3; x ++) {
+      board[x][y] = {
+        position: {x, y}
+      };
+    }
+  }
+  return {
+    players,
+    player_sides,
+    board,
+    turn: players[0]
```

Comment:
Do we want a state where we ask the players who goes first ? Or should we consider player ordering part of the game lobby job ?